homeassistant/components/vartastorage/__init__.py

```diff
@@ -0,0 +1,70 @@
+"""The VARTA Storage integration."""
+from __future__ import annotations
+
+import asyncio
+import logging
+
+from vartastorage.vartastorage import VartaStorage
+
+from homeassistant import config_entries, core
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryNotReady
+
+from .const import DOMAIN
+
+PLATFORMS: list[str] = ["sensor"]
+_LOGGER = logging.getLogger(__name__)
```

Comment:
Define `LOGGER` in `const.py` and import it here and in `sensor.py`

Reply:
done